src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -2,56 +2,68 @@
+| SoftwareVersions              | RECOMMENDED           | [string][]    | Manufacturer's designation of software version of the equipment that produced the composite instances. Corresponds to DICOM Tag 0018, 1020 `Software Versions`                                                                                                            |
+| HardcopyDeviceSoftwareVersion | [DEPRECATED][]        | [string][]    | Manufacturer's designation of the software of the device that created this Hardcopy Image (the printer). Corresponds to DICOM Tag 0018, 101A `Hardcopy Device Software Version`                                                                                           |
+| MagneticFieldStrength         | RECOMMENDED           |               | Nominal field strength of MR magnet in Tesla. Corresponds to DICOM Tag 0018,0087 `Magnetic Field Strength`                                                                                                                                                                |
+| ReceiveCoilName               | RECOMMENDED           |               | Information describing the receiver coil. Corresponds to DICOM Tag 0018, 1250 `Receive Coil Name`, although not all vendors populate that DICOM Tag, in which case this field can be derived from an appropriate private DICOM field                                      |
+| ReceiveCoilActiveElements     | RECOMMENDED           |               | Information describing the active/selected elements of the receiver coil. This doesn't correspond to a tag in the DICOM ontology. The vendor-defined terminology for active coil elements can go in this field. See an example below the table.                           |
+| GradientSetType               | RECOMMENDED           |               | It should be possible to infer the gradient coil from the scanner model. If not, e.g. because of a custom upgrade or use of a gradient insert set, then the specifications of the actual gradient coil should be reported independently                                   |
+| MRTransmitCoilSequence        | RECOMMENDED           |               | This is a relevant field if a non-standard transmit coil is used. Corresponds to DICOM Tag 0018, 9049 `MR Transmit Coil Sequence`                                                                                                                                         |
+| MatrixCoilMode                | RECOMMENDED           |               | (If used) A method for reducing the number of independent channels by combining in analog the signals from multiple coil elements. There are typically different default modes when using un-accelerated or accelerated (e.g. GRAPPA, SENSE) imaging                      |
```

Comment:
what datatype is this?

Reply:
This is unused, except strings containing a space (`" "`). It reads like a string, though.